src/utils.mm

```diff
@@ -0,0 +1,87 @@
+
```

Comment:
Files should be renamed using `git mv` to preserve modification history. Now it looks like the original file was deleted and you became the author of the whole file.

Reply:
I'll try to rebase and separate commits to preserve history, then force-push. It would be ok?

Reply:
Ok, I see, GitHub displays it properly when looking on a single commit but fails for the joined diff of the whole PR.